packages/forms-web-app/src/api/applications-download/controller.js

```diff
@@ -0,0 +1,15 @@
+const logger = require('../../lib/logger');
+const { getApplicationsCSV } = require('../_services/get-applications-csv');
+
+const getApplicationsDownload = async (req, res) => {
+	try {
+		const applicationsCSV = await getApplicationsCSV();
+
+		return res.attachment('applications.csv').send(applicationsCSV);
+	} catch (error) {
+		logger.error(error);
+		return res.status(500).send(error);
```

Comment:
I would send a more generic error message here, so as not to expose any internal error messages to the user